src/runtimes/index.ts

```diff
@@ -110,4 +112,28 @@ const getFunctionsFromPaths = async (
   return new Map(functionsWithConfig)
 }
 
-export { getFunctionsFromPaths }
+/**
+ * Gets a list of functions found in a list of paths.
+ */
+const getFunctionFromPath = async (
+  path: string,
+  { config, featureFlags = defaultFlags }: { config?: Config; featureFlags?: FeatureFlags } = {},
+): Promise<FunctionSource | undefined> => {
+  const fsCache = makeFsCache()
+
+  for (const runtime of RUNTIMES) {
+    // eslint-disable-next-line no-await-in-loop
```

Comment:
`getFunctionsFromPaths` implements this using `reduce`, but it's very hard to read. I'd prefer disabling the lint rule over the `reduce` implementation, really. How would you implement this (without parallelising it using `Promise.all`)?

Reply:
I think we should try to avoid disabling rules as much as possible, because that defeats the purpose of having them in the first place. Except for the ones in tests, I think all the cases where we're disabling a rule have a specific reason.

I personally would write it with a `reduce` in line with `getFunctionsFromPaths`, but if you think that's hard to read, perhaps we need to rethink the `no-await-in-loop` rule?

Reply:
let's rethink the rule, and keep the exception for now.